pnpm-lock.yaml

```diff
@@ -41875,39 +41998,6 @@ packages:
       yargs-parser: 21.1.1
     dev: true
 
-  /ts-jest/29.1.1_gvmt7aj7nai5bnvsxmiksfugce:
```

Comment:
neat that this gets us down to one copy of ts-jest. I'm not sure why this change removes the copy of jest without the ` '@babel/core': 7.24.4` dep but I'll take it.